ql/experimental/catbonds/catrisk.cpp

```diff
@@ -25,7 +25,7 @@
 namespace QuantLib {
 
     namespace {
-        Integer round(Real r) {
+        Integer int_round(Real r) {
             return (r > 0.0) ? Integer(std::floor(r + 0.5)) : Integer(std::ceil(r - 0.5));
         }
     }
```

Comment:
This looks like `std::lround`, which we didn't use before but is available now that we switched to C++11.

Reply:
That makes sense - we've just pushed that change.